### PR TITLE
TMA-824: Test filters created with .get_filters

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/748/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/609/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/870/ : SUCCESS in 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/749/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/610/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/871/ : SUCCESS in 33s

File: spec/integration/user_filters_spec.rb [hunk 1]
```diff
@@ -168,6 +168,18 @@
...
 
+  it 'works with filters from .get_filters' do
+    filters = GoodData::UserFilterBuilder.get_filters(
+      File.expand_path('../../data/user_filters.csv', __FILE__),
+      :type => :filter,
+      :labels => [{ :label => @label, :column => 'filter' }]
+    )
+    @project.add_data_permissions(filters)
+    permissions = @project.data_permissions
+    pretty = permissions.first.pretty_expression
```
Comment (kubamahnert) [inline spec/integration/user_filters_spec.rb]: does this have to be order dependent? wouldn't it be safer to check if the count of the permissions is 2 and one of them is tomas@gooddata.com ?

**Comment by kubamahnert** [comment]
LGTM!

**Comment by kubamahnert** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/292/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/635/ : SUCCESS in 41s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/896/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1222/ : SUCCESS in 53m 36s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/980/ : UNSTABLE in 2m 26s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/295/ : SUCCESS in 32s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/638/ : SUCCESS in 1m 54s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/899/ : SUCCESS in 1m 53s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1226/ : FAILURE in 2m 44s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/984/ : SUCCESS in 37m 51s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/299/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/642/ : SUCCESS in 3m 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/903/ : SUCCESS in 3m 43s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1230/ : FAILURE in 51m 59s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/988/ : UNSTABLE in 18m 39s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/303/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/651/ : SUCCESS in 58s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/912/ : SUCCESS in 59s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1234/ : FAILURE in 49m 04s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/992/ : UNSTABLE in 29m 27s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/304/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/654/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/915/ : SUCCESS in 34s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1235/ : FAILURE in 48m 47s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/993/ : UNSTABLE in 11m 46s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/309/ : SUCCESS in 1m 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/661/ : SUCCESS in 1m 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/922/ : SUCCESS in 1m 50s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1247/ : SUCCESS in 53m 25s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1003/ : SUCCESS in 40m 24s

**Pull request merged**